definitions/routing.lua

```diff
@@ -0,0 +1,249 @@
+-- move all of these down into rc config
+--
+-- or should these be put in routing lib?
+local TRACK_INFO_AUDIO_SRC_DISABLED = -1
+local TRACK_INFO_MIDIFLAGS_ALL_CH = 0
+local TRACK_INFO_MIDIFLAGS_DISABLED = 4177951
+local TRACK_INFO_CATEGORY_SEND = 0 -- send
+local TRACK_INFO_CATEGORY_RECIEVE = -1 -- send
+local TRACK_INFO_CATEGORY_HARDWARE = 1 -- send
+
+--//////////////////////////////////////////////////
+--
+-- ROUTING
+--
+--  neither of a and m | a and not m
+--      create AUDIO send (default)
+--
+--  only m
+--      create midi send
+--
+--  a and m
+--      create both audio and m
+--
+--  ----------------------------
+--
+--  the param_name is not necessarily the input format ?!
```

Comment:
if comments are to be written it's best to imagine you are writing for a 5 year old because that's simple information is more digestable